libbeat/publisher/pipeline/pipeline.go

```diff
@@ -151,6 +153,7 @@ type queueFactory func(queue.Eventer) (queue.Queue, error)
 // queue and outputs will be closed.
 func New(
 	beat beat.Info,
+	monitors Monitors,
```

Comment:
Todo: monitors contains logger, metrics + telemetry registry. Why still pass 'metrics', is this a different instance?